### PR TITLE
Adds Publish action to RabbitMQ

### PR DESCRIPTION
**Pull request by jtopjian**
This commit adds a publish action to the RabbitMQ pack, allowing a crafted message to be published to a RabbitMQ server.

### CODE REVIEW

File: packs/rabbitmq/actions/publish_message.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from kombu import BrokerConnection, Exchange, Queue
```
Comment (manasdk) [inline packs/rabbitmq/actions/publish_message.py]: This pack is built around pika. Using kombu probably works as it is a core stackstorm dependency but it is perhaps better to use pika.

Something like https://www.rabbitmq.com/tutorials/tutorial-one-python.html

Pika generally makes for simpler APIs than kombu. In core stackstorm also we are considering a move towards pika.

Comment (jtopjian) [inline packs/rabbitmq/actions/publish_message.py]: Yeah, I chose Kombu since it's required by StackStorm. I'll take a look at switching over to Pika.

**Comment by jtopjian** [comment]
@manasdk switched to Pika. Let me know how this looks. I can squash the commits to something cleaner before the final merge.

**Comment by manasdk** [comment]
Looks good. Thanks for switching to Pika.

Travis is complaiing about line length -
/home/travis/build/StackStorm/st2contrib/packs/rabbitmq/actions/publish_message.py:6:101: E501 line too long (129 > 100 characters)

Will merge after that gets fixed.

**Comment by jtopjian** [comment]
@manasdk looks like everything is checking out. Let me know if there are any other changes to make.

**Comment by manasdk** [comment]
Done. Merged!

**Comment by Kami** [comment]
Nice :+1:

**Pull request merged**